code/datums/gamemode/factions/vox_shoal.dm

```diff
@@ -106,14 +106,15 @@ var/list/potential_bonus_items = list(
 
 	for (var/i = 1 to 4)
 		var/chosen_one = pick(potential_bonus_items_temp)
-		potential_bonus_items_temp =- chosen_one
```

Comment:
lol

Reply:
this is funny. I also did a grep to see if this happened somewhere else and the answer is yes:

https://github.com/vgstation-coders/vgstation13/blob/94e4ee25da2bbd1eb53aff5a4a26d0b19cfda7e6/code/modules/events/meteors.dm#L234

Reply:
This should really runtime anytime the var on the right isn't a number
I haven't checked, though, so maybe it does